src/serialize.ts

```diff
@@ -48,6 +48,12 @@ export default function serialize(
     return `a:${serializeObject(item, scope)}`
   }
 
+  if (item instanceof Map) {
+    return `a:${item.size}:{` + Array.from(item.entries()).map(([value, key]) => {
+      return `${serialize(value, scope)}${serialize(key, scope)}`
```

Comment:
I think key and value are switched.
```suggestion
    return `a:${item.size}:{` + Array.from(item.entries()).map(([key, value]) => {
      return `${serialize(key, scope)}${serialize(value, scope)}`
```

Reply:
The naming is wrong but the output is correct, I'll update the naming in `main`

Reply:
oops